app/assets/scripts/views/owner.js

```diff
@@ -32,12 +32,14 @@ var Owner = React.createClass({
 
     const lang = this.props.meta.lang;
     const ownerProjects = projects.filter((project) => {
-      const name = project.local_manager[lang];
-      const sluggedName = slugify(name);
-      if (sluggedName === ownerName) {
-        ownerDisplayName = name;
+      let name = lang === 'en' ? project.local_manager : project.local_manager_ar || project.local_manager;
```

Comment:
I'm pretty sure this would show no `ownerProjects` for arabic pages, since `ownerName`, which you've written the path for below, is always picking the english variant.

We always use English names in the path and, therefore, as the filter key for related projects.